### ISSUE TITLE
ux: Website / Explaination

### ISSUE THREAD

**Issue by ElectricCookie** [lesspass/lesspass#218]
Hey there, 
*I've been looking for an alternative to my current setup (KeePass) and stumbled on this project.* 

Looking at the website and the first paragraph of it I initially thought the software was advertised as not needing to sync because it is web based. This was initially a turn off for me since I didn't know the passwords essentially never leave your PC. 

I think the average user / lazy reader needs to be directly confronted with this idea in a very simple way. 

The websites states the following: 

> LessPass derives a site, a login and a master password to generate a unique password. You don't need to sync your password vault across every device.

Though something like this would be way easier to understand (especially for people that don't know how encryption/ deriving works:

> LessPass computes your password using a master password and your username. Your password doesn't even leave your device and is never stored. You can even use LessPass without an active internet connection.

I understand that all this information and a way better explanation is given in the medium article which is linked under "how it works", but that is already 1 click away.

**Comment by guillaumevincent** [comment]
@ElectricCookie Thank you, can we iterate in this issue to create the best sentence?

> LessPass computes your password using a master password and your username. 

Ok for `computes` instead of `derives`+`generate`

> Your password doesn't even leave your device and is never stored. You can even use LessPass without an active internet connection.

two times `event` sounds weird. I'm not a native English speaker btw. So maybe I'm wrong.

> You can even use LessPass without an active internet connection.

 * And LessPass works offline
 * And LessPass works without an active internet connection

Any help on this one @derelk or @Facel3ss1?
Thanks

**Comment by ElectricCookie** [comment]
Wow very active maintenance! I'm impressed!
I left out 'site' by accident, and my proposal was by no means intended for publishing .

How about this:

> LessPass uses cryptographic methods to generate a safe password from username, site and your master password. It works without ever storing or transmitting any of your credentials over the internet.

**Comment by guillaumevincent** [comment]
> especially for people that don't know how encryption/ deriving works

So I prefer `LessPass computes unique password using a site, login and master password` instead of  `LessPass uses cryptographic methods to generate a safe password from username, site and your master password`.

`safe password` is subjective, `unique password` is a fact

`You don't need to sync your password vault across every device.` I like this sentence because the product is made for people who come from KeePass. We need to find a sentence that emphasizes the fact that we do not need synchronization.

**Comment by panther2** [comment]
So why not simply say:
`You don't need to sync your password vault across every device. Actually, synchronization is not needed at all!`

**Comment by Facel3ss1** [comment]
@guillaumevincent @ElectricCookie 
I've looked at all your suggestions and I've come up with this:

> LessPass computes a unique password using a master password, login, and the site your password is for. You don't need to sync a password vault across every device. LessPass even works offline!

Your previous suggestions just say 'site'. This isn't clear to the reader what the site is for. So I expanded it to clarify what the site is for.

@panther2 I do see what you are trying to say in your suggestion, but exclamation marks are a bit too informal IMO. That's why I didn't include it.

**Comment by panther2** [comment]
@Facel3ss1 
No problem. I just wanted to comply with @guillaumevincent 's point 
> We need to find a sentence that emphasizes the fact that we do not need synchronization.
To me this appears to be still unsaid now. Though I do like your sentences so far.

**Comment by guillaumevincent** [comment]
Thank you,

@Facel3ss1 we can help the comprehension with an image and some number?

> You don't need to sync a password vault across every device because LessPass works offline!

![screenshot from 2017-05-11 18-28-22](https://cloud.githubusercontent.com/assets/943762/25960645/ce834b9e-3677-11e7-86d4-fafe641d2b38.png)

**Comment by Facel3ss1** [comment]
Yes, numbers would help greatly! 😄
I'm presuming that picture is just a mockup because the red doesn't look that good...

Also, I really like that sentence too! 🎉

**Comment by guillaumevincent** [comment]
yes it's just a mockup :)

**Comment by ElectricCookie** [comment]
Maybe an illustration would be even better. Perhaps something that is a little more abstract than the interface.

**Comment by guillaumevincent** [comment]
@ElectricCookie like this one

![https://cdn-images-1.medium.com/max/2000/1*2NF5yLk018WAo0NF1KaBhQ.png]

**Comment by ElectricCookie** [comment]
Yes, but maybe a little less techy. This is already pretty indepth on the technology behind it.
